docs/src/modules/zend.cache.storage.adapter.rst

```diff
@@ -746,26 +746,26 @@ The Redis Adapter
-   |server              |                      |                         | Server can be described as follows:                                                  |
-   |                    |                      |                         |  - URI:   /path/to/sock.sock                                                         |
-   |                    |                      |                         |  - Assoc: array('host' => <host>[, 'port' => <port>[, 'timeout' => <timeout>]])      |
-   |                    |                      |                         |  - List:  array(<host>[, <port>, [, <timeout>]])                                     |
-   +--------------------+----------------------+-------------------------+--------------------------------------------------------------------------------------+
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |Name                |Data Type             |Default Value            |Description                                                                                   |
+   +====================+======================+=========================+==============================================================================================+
+   |database            |``integer``           |0                        |Set database identifier                                                                       |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |lib_option          |``array``             |``[]``                   | Associative array of redis options were the array key is the option name                     |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |namespace_separator |``string``            |":"                      |A separator for the namespace and prefix                                                      |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |password            |``string``            |""                       |Set password                                                                                  |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |persistent_id       |``string``            |````                     |Set persistent id (*name* of the connection, [read more](http://redis.io/topics/persistence)) |
```

Comment:
This is exactly the point of the PR. Actually it's specified that persistent_id could be `RDB` or `AOF`, while this is something related to server configuration. So this way user understand that the persistent_id should be only a generic string, while the control of the persistence is done via server (and if they want to know more about this, they can check the docs about Redis Persistence).